src/translations/screens/subscreens/TravelAidSettingsTexts.ts

```diff
@@ -0,0 +1,69 @@
+import {translation as _} from '../../commons';
+
+const TravelAidSettingsTexts = {
+  header: {
+    accessibility: {
+      title: _('Tilgjengelighet', 'Accessibility', 'Tilgjengelegheit'),
+    },
+    howTo: {
+      title: _('Viktig å vite', 'Important information', 'Viktig å vite'),
+    },
+  },
+
+  toggle: {
+    title: _('Reisehjelp', 'Journey Aid', 'Reisehjelp'),
+    subText: _(
+      'Reisehjelp er en forenklet modus spesielt tilrettelagt for deg som trenger ekstra støtte når du reiser.\n\n* Stopp en buss eller trikk med appen, fra holdeplassen der du skal på.\n* Raskere tilgang på sanntids-oppdateringer om ankomst og neste holdeplass.\n* Snarvei til gyldige billetter.',
+      'Journey Aid is a simplified mode designed specifically for those who need extra support when traveling.\n\n* Use the app to signal a stop for a bus or tram from your boarding stop.\n* Get faster access to real-time updates on arrivals and upcoming stops, as well as a shortcut to your tickets.* Shortcut to valid tickets.',
+      'Reisehjelp er ein forenkla modus spesielt tilrettelagt for deg som treng ekstra støtte når du reiser.\n\n* Stopp ein buss eller trikk med appen, frå haldeplassen der du skal på.\n* Raskare tilgang på sanntids-oppdateringar om ankomst og neste haldeplass.\n* Snarveg til gyldige billettar.',
```

Comment:
I'm a little hesitant about adding these texts in this PR. The texts isn't in sync with what features we have in the app now, and it mentions more than one of them. For example both blue button, and shortcut to my tickets, which are two features that may be added in different phases.

I think it would be more safe to always have the text in the app in sync with the features in the app. Adding texts doesn't take much time, so there is no real benefit to pre-adding text about features that will come in the future.